index.html

```diff
@@ -0,0 +1,1380 @@
+                Face Added</h2>
+            </article>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="news-subscribe border border-left-0 border-right-0 py-4 pb-5 px-sm-2 width-custom">
+      <h6 class="main-color font-weight-bolder">SUBSCRIBE</h6>
+      <div class="row px-0 mx-0">
+        <div class="col-12 col-lg-8 pl-0">
+          <div class="jumbotron position-relative mr-0 py-0 pb-3 py-md-5 mb-0 d-flex justify-content-start justify-content-md-center">
+            <div class="position-relative subsc-img-cont">
+              <img src="./media/image/subsc(2).jpg" alt="news week image">
+              <img class="position-absolute mt-4" src="./media/image/subsc(1).jpg" alt="news week image">
+            </div>
+            <div class="d-flex flex-column ml-4 pl-4 align-self-md-center align-self-end subsc-txt-cont align-items-center align-items-md-start">
+              <h4 class="font-weight-bolder text-body mb-0 mb-md-4">In-depth Stories You Don't See Elsewhere About Topics You
+                Want to Read!</h4>
+              <p>All Access + Weekly Delivery.</p>
+              <p>A Membership That's Perfect for You</p>
+              <button
+                class="btn btn-default align-self-center align-self-md-start text-uppercase main-bg font-weight-bolder text-white mt-4"
+                type="button">subscribe &#10095;</button>
+            </div>
+          </div>
+        </div>
+
+        <h6 class="main-color font-weight-bolder d-block d-lg-none mt-4 mb-2">NEWS LETTER</h6>
+        <div class="pl-3 pl-md-0 col-12 col-lg-4">
+          <div class="news-letter border row  pl-0 ml-md-1">
+            <div class="col-3 col-lg-6 d-none d-md-flex px-0">
+              <img class="w-100" src="./media/image/newsl.png" alt="news week image">
+            </div>
+            <div class="col-12 col-md-9 col-lg-6 pr-0 pt-4 d-flex flex-column justify-content-md-center align-items-center align-items-md-start">
+              <h4 class="font-weight-bolder">The Starting 5</h4>
+              <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+              <form class="mt-4 w-100 d-flex flex-column" action="#">
+                <input class="" type="email" name="email" id="email-1" placeholder="email address">
+                <button class="btn main-bg text-white mt-3 p-1 text-uppercase font-weight-bolder" type="submit">free sign
+                  up &#10095;</button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <footer class="container-fluid mt-3">
+      <div class="footer-header d-felx row border-bottom pb-3">
+        <div class="footer-logo col-12 col-md-8 d-flex flex-column flex-md-row">
+          <img class="footer-logo" src="./media/image/header-logo1.png" alt="news week image">
```

Comment:
- [ ] **[MANDATORY]** Please, add a bottom padding to your footer, text too close to the borders makes the rest of your nice work look unprofessional